crates/nu-cmd-extra/src/extra/bits/into.rs

```diff
@@ -98,13 +97,6 @@ impl Command for BitsInto {
                     Span::test_data(),
                 )),
             },
-            Example {
-                description: "convert a datetime value into a string, padded to 8 places with 0s",
-                example: "2023-04-17T01:02:03 | into bits",
```

Comment:
I'm fine with this, but instead of removing this example completely, maybe it's better to just change it to `2023-04-17T01:02:03 | format date '%c' | into bits`. Otherwise, if they try to convert a date, it will fail, and they'll be lost and dig in the documentation never to figure out how.

Reply:
Honestly I think keeping this example would be introducing tech-debt in the documentation. This conversion which is still in `into binary` is questionably usable.
I would like to remove it from `into binary` as well, but think that there is an impetus to deprecating properly (tricky with the current `operate` step) as folks may be accidentally piping `date` into it.
Here I don't think anyone uses `into bits` in earnest expecting this output and having both a new type error and a release note about this should be more than enough.